src/components/delivery/Compass.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { selectRoverPosition } from "../../store/telemetrySlice";
+import "./Compass.css";
+
+const Compass = () => {
+  const { orientW, orientX, orientY, orientZ } = useSelector(selectRoverPosition);
+  // Calculate the direction based on the quaternion values
+  const direction = Math.round(
+    (Math.atan2(
```

Comment:
Can you add a comment linking to where you got this math from? Makes reading the code easier.